packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -142,6 +169,27 @@ export class AtomicQuickviewModal implements InitializableComponent {
     return !!this.content && !!this.result;
   }
 
+  private handleKeywordHighlight() {
+    const doc = this.iframeRef?.contentWindow?.document;
+    if (!doc) {
+      return;
+    }
+    if (this.highlightKeywords) {
+      doc.getElementById('CoveoDisableHighlightStyle')?.remove();
+    } else {
+      const head = doc.head;
+      const style = doc.createElement('style');
+      style.setAttribute('id', 'CoveoDisableHighlightStyle');
+      head.appendChild(style);
+      style.appendChild(
+        doc.createTextNode(`[id^=CoveoHighlight] {
```

Comment:
It looks like this should be:
```suggestion
        doc.createTextNode(`[id^="${HIGHLIGHT_PREFIX}"] {
```